vrecord

```diff
@@ -541,9 +541,8 @@ if [[ "${runtype}" = "audiopassthrough" ]] ; then
 #[f1][abcde1]hstack[abcdef1],\
 #[abcdef1][xx]overlay=200:10[vo]"
 #            ;;
-#        Q) exit 0 ;;
-#        q) exit 0 ;;
-#        *)  report -w "Error: Not a valid option, select a valid number or [q] to quit." ; return 1 ;;
+#        "quit") report -d "Bye then" ; exit 0 ;;
+#        *) report -w "Error: Not a valid option, select a valid number." ; return 1 ;;
```

Comment:
report is defined at https://github.com/amiaopensource/vrecord/blob/f22cfc811b31a2ccc1adbe65c3c49c623d748715/vrecord#L253